ding/worker/collector/tests/speed_test/test_collector_profile_cartpole.py

```diff
@@ -0,0 +1,166 @@
+import logging
+import time
+import copy
+import pytest
+import numpy as np
+from easydict import EasyDict
+from functools import partial
+
+from ding.worker import SampleSerialCollector, NaiveReplayBuffer
+from ding.envs import get_vec_env_setting, create_env_manager, AsyncSubprocessEnvManager, SyncSubprocessEnvManager,\
+    BaseEnvManager, get_env_manager_cls
+from ding.utils import deep_merge_dicts, set_pkg_seed, pretty_print
+
+from ding.envs import BaseEnvManager, DingEnvWrapper
+from ding.policy import DQNPolicy
+from ding.model import DQN
+import gym
+from ding.rl_utils import get_epsilon_greedy_fn
+
+
+# Get DI-engine form env class
+def wrapped_cartpole_env():
+    return DingEnvWrapper(gym.make('CartPole-v0'))
+
+
+env_policy_cfg_dict = dict(
+    cartpole_dqn=dict(
+        size='cartpole_dqn',
+        env=dict(
+            collector_env_num=8,
+            stop_value=195,
+            reset_time=0.5,
+        ),
+        policy=dict(
+            cuda=False,
+            model=dict(
+                obs_shape=4,
+                action_shape=2,
+                encoder_hidden_size_list=[128, 128, 64],
+                dueling=True,
+            ),
+            collect=dict(
+                n_sample=80,
+                collector=dict(collect_print_freq=1000000),
+            ),
+            other=dict(
+                eps=dict(
+                    type='exp',
+                    start=0.95,
+                    end=0.1,
+                    decay=10000,
+                ),
+                replay_buffer=dict(replay_buffer_size=20000, ),
+            ),
+        ),
+    )
+)
+
+# SLOW MODE: used in normal test
+#   - Repeat 3 times; Collect 300 times;
+#   - Test on small + middle + big env
+#   - Test on base + async_subprocess + sync_subprocess env manager
+#   - Test with reset_ratio = 1 and 5.
+# FAST MODE: used in CI benchmark test
+#   - Only once (No repeat); Collect 50 times;
+#   - Test on small env
+#   - Test on base + sync_subprocess env manager
+#   - Test with reset_ratio = 1.
+FAST_MODE = True
+if FAST_MODE:
+    # Note: 'base' takes approximately 6 times longer than 'subprocess'
+    #test_env_manager_list = ['base', 'subprocess', 'gym3_async']
+    test_env_manager_list = ['base', 'subprocess', 'async_subprocess', 'gym_async']
```

Comment:
Could we have a detailed benchmarking results doc for all env managers in the future?